lib/broadway.ex

```diff
@@ -681,6 +681,7 @@ defmodule Broadway do
   Returned by `start_link/2`.
   """
   @type on_start() :: {:ok, pid()} | :ignore | {:error, {:already_started, pid()} | term()}
+  @type via_tuple :: {:via, Registry, any()}
```

Comment:
```suggestion
  @type via_tuple :: {:via, module, any()}
```

Reply:
Or maybe even better:

```suggestion
  @type name :: atom() | {:via, module(), term()}
```

And then we can reuse it throughout the API?